### DISCUSSION TITLE
InsertMany into table Oracle

### DISCUSSION THREAD

**Discussion by gianluca13g** [sqlkata/querybuilder#645] [General]
Hey there,

I found a issue that it's not possible to insert many items into table (Oracle SQL) because the syntax that SqlKata generated isn't right for oracle query.

Thanks,
Gianluca
![sqldeveloper64W_UVBZDJYQE7](https://user-images.githubusercontent.com/99675492/205030066-ee9cd0a0-b858-4ae3-98ad-4a45be483c74.png)

**Comment by andrewhosack** [comment]
Try downloading the prerelease, this issue has been resolved.
